Soccar [UI]/src/nl/soccar/ui/input/Keyboard.java

```diff
@@ -62,29 +63,19 @@
     /**
      * Constructor
      */
-    private Keyboard() {
-    }
-
-    /**
-     * Method that checks if the given key is already present in pressedKeys
-     * list.
-     *
-     * @param code The keycode that needs to be checked.
-     * @return boolean True if already pressed, false if not in pressedKeys
-     * list.
-     */
-    public static boolean isPressed(KeyCode code) {
-        return PRESSED_KEYS.contains(code);
+    public Keyboard() {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1186)